optimade/server/routers/utils.py

```diff
@@ -91,20 +91,17 @@ def meta_values(
     )
 
 
-def handle_response_fields(
+def remove_exclude_fields(
```

Comment:
Is the removal of `handle_response_fields` really necessary? Again, my server code uses this independently of this package. We need to be careful not to force people into a rewrite of their existing stuff for the sake of new features (that they may or may not use)